packages/design-system/src/lib/components/form/form-group/form-element/FormInput.tsx

```diff
@@ -26,6 +27,7 @@ export const FormInput = React.forwardRef(function FormInput(
     value,
     required,
     autoFocus,
+    autoComplete,
```

Comment:
Just curious why autoComplete was added here? I see that it is a property in SearchInput.tsx, but it is set to "off".

Reply:
Yes, this is because the current `FormInputProps` interface wasn't allowing me to pass the native attribute autoComplete, so I explicitly declared it.
I am using it in the search input with an "off" value to avoid the browser to provide automated assistance in filling out the search input.
I think it's a better experience without it.